.github/workflows/ci.yaml

```diff
@@ -1,6 +1,12 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
2023? :-D

Reply:
Normally it should represent the current year. No need to change it here (now). I will check official Canonical policy first.